### ISSUE TITLE
[Nextcloud 15] Unable to generate a URL for the named route "accessibility.accessibility.getJavascript" as such route does not exist, undefined index showgridview, isIE

### ISSUE THREAD

**Issue by flokli** [nextcloud/server#13094]
### Steps to reproduce
1. Upgrade to Nextcloud 15.0.0
2. Observe logs while accessing pages

### Expected behaviour
Tell us what should happen

### Actual behaviour
Tell us what happens instead
Nextcloud complains the following:
```
{no app in context} {"Exception":"Symfony\\Component\\Routing\\Exception\\RouteNotFoundException","Message":"Unable to generate a URL for the named route \"accessibility.accessibility.getJavascript\" as such route does not exist.","Code":0,"Trace":[{"file":"\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/lib\/private\/Route\/Router.php","line":337,"function":"generate","class":"Symfony\\Component\\Routing\\Generator\\UrlGenerator","type":"->","args":["accessibility.accessibility.getJavascript",{"v":"0"},1]},{"file":"\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/lib\/private\/Route\/CachingRouter.php","line":60,"function":"generate","class":"OC\\Route\\Router","type":"->","args":["accessibility.accessibility.getJavascript",{"v":"0"},false]},{"file":"\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/lib\/private\/URLGenerator.php","line":79,"function":"generate","class":"OC\\Route\\CachingRouter","type":"->","args":["accessibility.accessibility.getJavascript",{"v":"0"}]},{"file":"\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/apps\/accessibility\/lib\/AppInfo\/Application.php","line":70,"function":"linkToRoute","class":"OC\\URLGenerator","type":"->","args":["accessibility.accessibility.getJavascript",{"v":"0"}]},{"file":"\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/apps\/accessibility\/appinfo\/app.php","line":31,"function":"injectJavascript","class":"OCA\\Accessibility\\AppInfo\\Application","type":"->","args":[]},{"file":"\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/lib\/private\/legacy\/app.php","line":261,"args":["\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/apps\/accessibility\/appinfo\/app.php"],"function":"require_once"},{"file":"\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/lib\/private\/legacy\/app.php","line":154,"function":"requireAppFile","class":"OC_App","type":"::","args":[{"__class__":"OCA\\Accessibility\\AppInfo\\Application"}]},{"file":"\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/lib\/private\/legacy\/app.php","line":127,"function":"loadApp","class":"OC_App","type":"::","args":["accessibility"]},{"file":"\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/lib\/base.php","line":972,"function":"loadApps","class":"OC_App","type":"::","args":[]},{"file":"\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/index.php","line":42,"function":"handleRequest","class":"OC","type":"::","args":[]}],"File":"\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/3rdparty\/symfony\/routing\/Generator\/UrlGenerator.php","Line":113,"CustomMessage":"--"}
Nextcloud[16588]: {PHP} Undefined index: isIE at /nix/store/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0/apps/files/templates/index.php#5
Nextcloud[16588]: {PHP} Undefined index: showgridview at /nix/store/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0/apps/files/templates/index.php#7
Nextcloud[16588]: {PHP} Undefined index: showgridview at /nix/store/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0/apps/files/templates/index.php#
```

This seems to be a similar issue like https://github.com/nextcloud/server/pull/5821 - cc @MorrisJobke 

### Server configuration

**Operating system**: NixOS 18.09

**Web server:** nginx-1.14.1

**Database:** mariadb-10.2.17

**PHP version:** php-7.1.25

**Nextcloud version:** (see Nextcloud admin page) 15.0.0

**Updated from an older Nextcloud/ownCloud or fresh install:** 14.0.4

**Where did you install Nextcloud from:** https://download.nextcloud.com/server/releases/nextcloud-15.0.0.tar.bz2

**Comment by MorrisJobke** [comment]
The isIE should be fixed by https://github.com/nextcloud/server/pull/12385 but this is shipped in 15.0.0. 🤔 

Could you check `occ app:list` and post the output here? This maybe shows if the needed apps are correctly enabled.

**Comment by flokli** [comment]
I couldn't see it current logs anymore, last occurence Dec 16th:

```
Dec 16 12:50:44 gorleben systemd[1]: Starting nextcloud-cron.service...
Dec 16 12:50:51 gorleben Nextcloud[16955]: {no app in context} {"Exception":"Symfony\\Component\\Routing\\Exception\\RouteNotFoundException","Message":"Unable to generate a URL for the named route \"accessibility.accessibility.getJavascript\" as such route does not exist.","Code":0,"Trace":[{"file":"\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/lib\/private\/Route\/Router.php","line":337,"function":"generate","class":"Symfony\\Component\\Routing\\Generator\\UrlGenerator","type":"->","args":["accessibility.accessibility.getJavascript",{"v":"0"},1]},{"file":"\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/lib\/private\/Route\/CachingRouter.php","line":60,"function":"generate","class":"OC\\Route\\Router","type":"->","args":["accessibility.accessibility.getJavascript",{"v":"0"},false]},{"file":"\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/lib\/private\/URLGenerator.php","line":79,"function":"generate","class":"OC\\Route\\CachingRouter","type":"->","args":["accessibility.accessibility.getJavascript",{"v":"0"}]},{"file":"\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/apps\/accessibility\/lib\/AppInfo\/Application.php","line":70,"function":"linkToRoute","class":"OC\\URLGenerator","type":"->","args":["accessibility.accessibility.getJavascript",{"v":"0"}]},{"file":"\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/apps\/accessibility\/appinfo\/app.php","line":31,"function":"injectJavascript","class":"OCA\\Accessibility\\AppInfo\\Application","type":"->","args":[]},{"file":"\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/lib\/private\/legacy\/app.php","line":261,"args":["\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/apps\/accessibility\/appinfo\/app.php"],"function":"require_once"},{"file":"\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/lib\/private\/legacy\/app.php","line":154,"function":"requireAppFile","class":"OC_App","type":"::","args":[{"__class__":"OCA\\Accessibility\\AppInfo\\Application"}]},{"file":"\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/lib\/private\/legacy\/app.php","line":127,"function":"loadApp","class":"OC_App","type":"::","args":["accessibility"]},{"file":"\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/ocs\/v1.php","line":56,"function":"loadApps","class":"OC_App","type":"::","args":[]},{"file":"\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/ocs\/v2.php","line":24,"args":["\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/ocs\/v1.php"],"function":"require_once"}],"File":"\/nix\/store\/r963kb3s1c7xwsrgsf4dvcc6v4rqdj44-nextcloud-15.0.0\/3rdparty\/symfony\/routing\/Generator\/UrlGenerator.php","Line":113,"CustomMessage":"--"}
Dec 16 12:51:36 gorleben systemd[1]: Started nextcloud-cron.service.
Dec 16 13:05:45 gorleben systemd[1]: Starting nextcloud-cron.service...
Dec 16 13:06:15 gorleben systemd[1]: Started nextcloud-cron.service.
```

Note the subsequent `nextcloud-cron` run didn't yield any more errors - could it be this might have been somewhat cached files, not properly being purged during backup?

**Comment by MorrisJobke** [comment]
> could it be this might have been somewhat cached files, not properly being purged during backup?

Usually not, because it's cached inside the PHP opcache. Anyways - let's close it for now. We can reopen if it still happens.
